src/utils.py

```diff
@@ -0,0 +1,72 @@
+import atheris
+import numpy as np
+from sklearn.base import is_classifier
+from sklearn.datasets import load_iris, make_regression
+
+def _classification_training() -> (np.array, np.array, dict):
+    """
+    Create a classification training set
+    """
+    training_x, training_y = load_iris(return_X_y=True)
+    data_info = {
+        "dimensions": 4,
+        "min_feature": min([ min(minimum) for minimum in training_x]),
+        "max_feature": max([ max(maximum) for maximum in training_x]),
```

Comment:
I think you can do this without storing the whole lists:
```suggestion
        "min_feature": min(min(minimum) for minimum in training_x),
        "max_feature": max(max(maximum) for maximum in training_x),
```
I'm assuming `min` and `max` work over an iterable, but I may be remembering it wrong. I think it's worth trying.

Reply:
I tried the change and it works, adding it.

Reply:
https://github.com/lambdaclass/concrete-ml-fuzzer/pull/10/commits/222437dac1d46aa090c069e84377d5ecb95d53a1